Modules/mmapmodule.c

```diff
@@ -255,6 +256,192 @@ do {                                                                    \
 } while (0)
 #endif /* UNIX */
 
+#if defined(MS_WIN32) && !defined(DONT_USE_SEH)
+static DWORD
+filter_page_exception(EXCEPTION_POINTERS *ptrs, EXCEPTION_RECORD *record)
+{
+    *record = *ptrs->ExceptionRecord;
+    if (record->ExceptionCode == EXCEPTION_IN_PAGE_ERROR
+        || record->ExceptionCode == EXCEPTION_ACCESS_VIOLATION) {
+        return EXCEPTION_EXECUTE_HANDLER;
+    }
+    return EXCEPTION_CONTINUE_SEARCH;
+}
+
+static DWORD
+filter_page_exception_method(mmap_object *self, EXCEPTION_POINTERS *ptrs, EXCEPTION_RECORD *record)
+{
+    *record = *ptrs->ExceptionRecord;
+    if (record->ExceptionCode == EXCEPTION_IN_PAGE_ERROR
+        || record->ExceptionCode == EXCEPTION_ACCESS_VIOLATION) {
```

Comment:
```suggestion
    if (record->ExceptionCode == EXCEPTION_IN_PAGE_ERROR ||
        record->ExceptionCode == EXCEPTION_ACCESS_VIOLATION) {
```

PEP 7:
* When you break a long expression at a binary operator, the operator goes at the end of the previous line